packages/context-entries-link-traversal/lib/Keys.ts

```diff
@@ -19,3 +19,12 @@ export const KeysRdfResolveHypermediaLinks = {
     '@comunica/bus-rdf-resolve-hypermedia-links:annotateSources',
   ),
 };
+
+export const KeysExtractLinksExtractTree = {
```

Comment:
I decided to stay consistent with the current naming, it will changed with the PR to rename the package.